src/events_dispatcher.js

```diff
@@ -1,4 +1,5 @@
 ;(function() {
+  var _window = Pusher.runtime.getWindow();
```

Comment:
In this case we actually want the runtime global. So maybe we should have a `Pusher.Runtime.getGlobal()` where the browser implementation returns `window`?